### PR TITLE
feat: add event cont for day, month and year

### PR DESCRIPTION
**Pull request by edouard-lopez**
Result of contributing session with @Farsen976 

* Added a `<EventCount />` component that take a list of events
  * adjust label to 0, 1 or more events
* Updated `<SelectedEvents>` and `<YearSelector>` to use `<EventCount/>`

[event count.webm](https://github.com/scraly/developers-conferences-agenda/assets/1212392/2262973f-cf6c-40f0-a8c2-b2ee117a06cf)

### CODE REVIEW

File: page/src/components/EventCount/EventCount.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const getCountText = (count) => {
+    console.log(`count: ${count}`);
```
Comment (scraly) [inline page/src/components/EventCount/EventCount.jsx:2]: console.log are only used in dev/debug phase, isn't it?
Can you remove them please? :)

**Review by scraly — CHANGES_REQUESTED** [review]

**Comment by scraly** [comment]
@gouz can you review the PR please? :)

**Comment by scraly** [comment]
Hi @edouard-lopez and @Farsen976 thanks for your contributions ❤️
We've add some little things to adjust in the review.
Thx! :)

File: page/src/components/EventCount/EventCount.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    if (typeof count === 'undefined' || count === 0) return 'no event'
```
[the remark this answers is not in the document]
Comment (edouard-lopez) [reply page/src/components/EventCount/EventCount.jsx:2]: We are inside the `getCountText()` method, better being explicit about our conditions

[the remark this answers is not in the document]
Comment (gouz) [reply page/src/components/EventCount/EventCount.jsx:2]: as you want, but something like

```
const getCountText = (count) => {
  if (count) {
    return `${count} event${count > 1 ? "s" : ""}`;
  }
  return "no event";
};
```
is readable too ;)

[the remark this answers is not in the document]
Comment (edouard-lopez) [reply page/src/components/EventCount/EventCount.jsx]: I Like that better, thanks. See 4c1d5c2

File: page/src/components/EventCount/EventCount.jsx [hunk 3]
```diff
@@ -1,7 +1,11 @@
 const getCountText = (count) => {
-    if (typeof count === 'undefined' || count === 0) return 'no event'
-    if (count === 1) return '1 event'
-    return `${count} events`
+    const getCountText = (count) => {
+        if (count) {
+          const plural = count > 1 ? "s" : ""
```
Comment (gouz) [inline page/src/components/EventCount/EventCount.jsx:4]: const plural = count > 1 ? "s" : "";

**Review by gouz — APPROVED** [review]

File: page/src/components/EventCount/EventCount.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const getCountText = (count) => {
+    const getCountText = (count) => {
```
Comment (scraly) [inline page/src/components/EventCount/EventCount.jsx:2]: Is it normal these two lines `const getCountText = (count) => {`?

**Review by scraly — CHANGES_REQUESTED** [review]

File: page/src/components/EventCount/EventCount.jsx (hunk 4 above)
Comment (edouard-lopez) [reply page/src/components/EventCount/EventCount.jsx]: nope, I'm testing GitHub codespace, must I messed up something. If you want to test it, you need to serve the app with to avoir [502 Bad Gateway error](https://github.com/orgs/community/discussions/28563#discussioncomment-4880737):
```
npm run dev -- --host 0.0.0.0
```

**Comment by scraly** [comment]
Thanks, for me now it's top! :) 
And I'm impressed about the nb of tech events listed this year :) 
image

**Pull request merged**